R/betydb.R

```diff
@@ -159,11 +160,113 @@ betydb_search <- function(query = "Maple SLA", ..., include_unchecked = NULL){
 }
 
 betydb_GET <- function(url, args = list(), key = NULL, user = NULL, pwd = NULL, which, ...){
-  txt <- betydb_http(url, args, key, user, pwd, ...)
-  lst <- jsonlite::fromJSON(txt, simplifyVector = TRUE, flatten = TRUE)
+
+  api_version <- ifelse(grepl('/beta/api', url), 'beta', 'v0')
+  if(!exists('per_call_limit')) {
```

Comment:
actually, the whole reason I added the `per_call_limit` as a parameter rather than a fixed number is so that it can be set to a smaller number in the testing. So in the testing I make `per_call_limit <<- 20` a global variable ... 

5000 is a reasonable limit when the package is being used and 20 is sufficient for testing 

It is kind of hacky, but is the difference b/w tests that take seconds vs minutes (e.g. so that I can test with iterations of [0,1,n] and remainders [0,n].